components/gitpod-cli/cmd/open.go

```diff
@@ -31,7 +33,13 @@ var openCmd = &cobra.Command{
 		for _, fn := range args {
 			_, err := service.OpenFile(theialib.OpenFileRequest{Path: fn})
 			if err != nil {
-				log.Println(err)
+				viCmd := exec.Command("vi", fn)
```

Comment:
I'd prefer `syscall.Exec` ... 
If we keep `gp` in the loop, we have to handle signal forwarding ourselves.

Reply:
```suggestion
				err = syscall.Exec("vi", append([]string{"vi"}, args...), os.Environ())
```

note: this is untested